data/ecosystems/s/scroll.toml

```diff
@@ -2409,6 +3627,12 @@ url = "https://github.com/unifralabs/scroll-contract-verify-demo"
 [[repo]]
 url = "https://github.com/unifralabs/scroll-explorer-l2-indexer"
 
+[[repo]]
+url = "https://github.com/unlock-protocol/unlock"
```

Comment:
This can be added as a sub ecosystem.

Reply:
Removed and added to Subecosystem. Good catch.